### DISCUSSION TITLE
Error compiling LIS 7.5 on Discover (SLES15 - Milan nodes)

### DISCUSSION THREAD

**Discussion by prakrutkansara** [NASA-LIS/LISF#1566] [Q&A] [answered]
Hello,

I am trying to compile LIS 7.5 on the new Milan nodes (SLES15) on Discover but the compilation fails. Previously, I have successfully compiled LIS 7.3 and 7.4 on SLES12 processors. I'm following the Discover quick start guide (https://nasa-lis.github.io/LISF/discover_quick_start/discover_quick_start.html) and after logging in to Discover on Milan login node, here's how I'm compiling it:

```
$ module purge
$ module load lisf_7.5_intel_2023.2.1
$ git checkout origin/support/lisf-public-7.5
$ cd lis
$ ./configure
$ ./compile
``` 

The compilation stops at "mpicc -c  -fp-model precise -traceback -DIFC -DLINUX -DLIS_JULES -I./ -I../core/ -I../params/gfrac/SPORTDaily/ -I../params/gfrac/VIIRSDaily/ -I../surfacemodels/land/vic.4.1.1/ -I../surfacemodels/land/vic.4.1.1/physics/ -I../surfacemodels/land/vic.4.1.2.l/ -I../surfacemodels/land/vic.4.1.2.l/physics/ -I../surfacemodels/land/awral.6.0.0/physics/ ../surfacemodels/land/vic.4.1.2.l/physics/initialize_model_state.c".

Attached is the error message: 
[lis7.5_compilation_error.txt](https://github.com/user-attachments/files/16022163/lis7.5_compilation_error.txt)

Thank you!

**Comment by dmocko** [comment, marked as answer]
Hi @prakrutkansara 

We are aware of this issue on the SLES15 Milan Nodes.  We are still trying to resolve it,
and apologies for not documenting this issue just yet.  As a workaround:

Create a lis/make/user.cfg file containing:
  VIC.4.1.1: Off
  VIC.4.1.2: Off

Then within "lis/make", type "gmake realclean", then re-configure and re-compile.

If you really need to run VIC, please let us know, as there may be another solution.

**Reply by prakrutkansara** [reply]
Thank you David! That solved the issue and I am able to run LIS.
